lib/ansible/modules/cloud/kubevirt/kubevirt_scale_vmirs.py

```diff
@@ -15,12 +15,12 @@
 
 short_description: Scale up or down KubeVirt VMI ReplicaSet
 
-version_added: "2.5"
+version_added: "2.8"
 
 author: KubeVirt Team (@kubevirt)
 
 description:
-  - "Use Kubernets Python SDK to scale up or down KubeVirt
+  - "Use Openshift Python SDK to scale up or down KubeVirt
```

Comment:
Kubevirt can be deployed in both Kubernetes and Openshift cluster. Let's keep both here.